webapp/src/core.ts

```diff
@@ -347,8 +347,14 @@ export function promptAsync(options: PromptOptions): Promise<string> {
     options.onLoaded = () => {
         let dialogInput = document.getElementById('promptDialogInput') as HTMLInputElement;
         if (dialogInput) {
-            dialogInput.focus();
```

Comment:
Why remove the focus? It's used to highlight the entire default argument, making it easy to select all and delete.